public/i18n/en.yaml

```
@@ -240,6 +240,7 @@ command-palette:
     open-preferences: Open Preferences
     resource-aliases: Built-in Resource Aliases
     short-help: Type to search, e.g. "ns default" or "pod nginx"
+    all-namespaces: List resources from all namespaces
```

```suggestion
    all-namespaces: List resources from all Namespaces
```

Ignore this if we don't capitalize "Namespace" in Busola.